src/main/java/ru/hogwarts/school/repository/AvatarRepository.java

```diff
@@ -0,0 +1,13 @@
+package ru.hogwarts.school.repository;
+
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import ru.hogwarts.school.model.Avatar;
+
+import java.util.Optional;
+
+@Repository
+public interface AvatarRepository  extends JpaRepository<Avatar, Long> {
+    Optional<Avatar> findByStudentAvatar(Long studentId);
```

Comment:
Ошибка в названии метода
```suggestion
    Optional<Avatar> findByStudent_Id(Long studentId);
```